core/.planning/phases/06.1-execution-queue-management/06.1-02-PLAN.md

```
@@ -0,0 +1,494 @@
+     */
+    @Repository
+    interface ExecutionQueueRepository : JpaRepository<ExecutionQueueEntity, UUID> {
+
+        /**
+         * Claim pending execution requests for processing.
+         *
+         * Uses SKIP LOCKED to allow concurrent consumers:
+         * - Claimed rows are locked but other PENDING rows remain available
+         * - Prevents duplicate processing across instances
+         * - Non-blocking for unclaimed rows
+         *
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)